turborepo-tests/examples/turbo.json

```diff
@@ -0,0 +1,26 @@
+{
+  "extends": ["//"],
+  "pipeline": {
+    "test": {
+      "dependsOn": ["^build"],
+      "outputs": [],
+      "inputs": [
+        "./run-example.sh",
+
+        // TODO: model ../../examples as a workspace so we can wire up
+        // a workspace "dependency" and use that to trigger changes instead
+        // of relative paths.
+        "../../examples/**/*.ts",
+        "../../examples/**/*.tsx",
+        "../../examples/**/*.json",
+        "../../examples/**/*.js",
+        "../../examples/**/*.yaml",
```

Comment:
Unless I got these paths wrong (which I don't _think_ I did??) this doesn't actually work. I'm not seeing these in `expandedInputs` in the Run Summary.